### PR TITLE
Actions documentation : add @flush_entity action

### PR DESCRIPTION
**Pull request by flochntrl**

### CODE REVIEW

**Comment by orocla** [comment]
Thank you for your pull request.

It looks like this may be your first contribution to an Oro, Inc. open source project, in which case you'll need to sign a Contributor License Agreement (CLA) at http://www.orocrm.com/contributor-license-agreement/

If you've already signed the CLA, it's possible we don't have your GitHub username or you're using a different email address. GitHub uses the email address you set in your local Git configuration to associate commits with your GitHub account. Please sign the CLA again using the correct GitHub username and email address or see this help article on [setting the email on your git commits](https://help.github.com/articles/setting-your-email-in-git/).

Once you've signed the CLA, please allow for some time for the submission to be processed.

File: src/Oro/Component/Action/Resources/doc/actions.md [hunk 1]
```diff
@@ -263,6 +264,32 @@ OR
...
+ - entity - data will contain entity instance;
+
+**Configuration Example**
+```
+- @flush_entity:
+     entity: $.someEntity #flush entity stored in some attribute
+
+OR
+
+- @find_entity: $.someEntity
```
Comment (anyt) [inline src/Oro/Component/Action/Resources/doc/actions.md]: `find` should be replaced with `flush`

File: src/Oro/Component/Action/Resources/doc/actions.md [hunk 2]
```diff
@@ -263,6 +264,32 @@ OR
...
+
+OR
+
+- @find_entity: ~ #flush context entity
```
Comment (anyt) [inline src/Oro/Component/Action/Resources/doc/actions.md]: `find` should be replaced with `flush`

File: src/Oro/Component/Action/Resources/doc/actions.md (hunk 1 above)
Comment (flochntrl) [reply src/Oro/Component/Action/Resources/doc/actions.md]: :+1:

File: src/Oro/Component/Action/Resources/doc/actions.md (hunk 2 above)
Comment (flochntrl) [reply src/Oro/Component/Action/Resources/doc/actions.md]: :+1:

**Comment by anyt** [comment]
Merged, 
Thank you for the contribution!

**Pull request closed without merging**